R/fiveyeargwsum-render.R

```diff
@@ -104,14 +104,17 @@ createfiveyeargwsumPlot <- function(reportObject){
   # Add space to the top of the Y Axis
   plot_object <- RescaleYTop(plot_object)
 
+  #Add approval explanation label to the top of the plot
+  plot_object <- mtext(plot_object, text = "Displayed approval level(s) are from the source TS that statistics are derived from.", side=3, cex=0.6, line=0.1, adj=1, axes=FALSE)
+
   #Add Min/Max labels if we aren't plotting min and max
-  line <- 0.33
+  line <- 0.1
```

Comment:
These numbers I could see possibly being pulled into styles since they aren't really calculated on, though this one is modified as part of the loop so I don't know if it specifically would fit? The numbers within the mtext() gsplot call would probably make sense in styles though